src/app.js

```diff
@@ -217,21 +217,11 @@ export default function(app) {
           reusableNode = reusableChild[1]
         }
 
-        if (null == oldKey && null == newKey) {
-          patch(element, oldElement, oldChild, newChild)
-          j++
-          i++
-        } else if (null == oldKey && null != newKey) {
-          if (reusableElement) {
-            element.insertBefore(reusableElement, oldElement)
-            patch(element, reusableElement, reusableNode, newChild)
```

Comment:
@dodekeract The diff is a bit misleading as it gives the wrong impressing that I "removed" it. It was just squashed together with another equvalent combination of `oldKey` and `newKey` values. Check out https://github.com/hyperapp/hyperapp/pull/200#issuecomment-298674240